### PR TITLE
Address skin feedback

### LINKED ISSUE #15044: Text bar is in bold now?
Also the text cuts off at the bottom for me so I can't tell the difference between v's and y's, i's and j's, and probably some other letters too

### PR DESCRIPTION
**Pull request by neersighted**
Fixes #15044
Closes #15045
dreamseeker_2016-01-29_15-53-49
dreamseeker_2016-01-29_15-53-59
dreamseeker_2016-01-29_16-47-24

### CODE REVIEW

**Comment by tkdrg** [comment]
Much better.

**Comment by Incoming5643** [comment]
but what does the text look like

**Comment by tkdrg** [comment]
@Incoming5643 the last pic

**Comment by optimumtact** [comment]
FLAT DESIGN
:laughing: :rocket: :space_invader:

File: interface/skin.dmf [hunk 1]
```diff
@@ -789,49 +789,55 @@ window "infowindow"
   elem "changelog"
 		type = BUTTON
 		button-type = pushbutton
-		pos = 0,0
-		size = 100x20
-		is-flat = true
+    pos = 16,5
```
Comment (MrStonedOne) [inline interface/skin.dmf]: This is space indenting (and further down) should be tabs like the rest.

**Comment by MrStonedOne** [comment]
Wouldn't it be better (and cleaner) to do just a flat border line rather then the old and outdated raised shadow effect?

Just do `border = line`

**Comment by Shadowlight213** [comment]
Still preferred the original pre flatten design. But this is better than the current monstrosity.

**Pull request merged**